src/main/java/salesforce/gui/pages/IndividualListPage.java

```diff
@@ -0,0 +1,102 @@
+/**
+ * Copyright (c) 2021 Fundacion Jala.
+ * This software is the confidential and proprietary information of Fundacion Jala
+ * ("Confidential Information"). You shall not disclose such Confidential
+ * Information and shall use it only in accordance with the terms of the
+ * license agreement you entered into with Fundacion Jala
+ */
+
+package salesforce.gui.pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+
+public class IndividualListPage extends BasePage {
+
+    private By recentlyViewedSpan = By.cssSelector("span.triggerLinkText");
+    private By deletedSuccessMessage = By.xpath("//span[contains(.,\"was deleted.\")]");
+    private WebElement dropDownMenu;
+
+    /**
+     * Checks if the view list span is visible.
+     *
+     * @return boolean
+     */
+    public boolean recentlyViewedSpanVisible() {
+        return findElement(recentlyViewedSpan).isDisplayed();
+    }
+
+    /**
+     * Deletes the last created or modified record.
+     */
+    public void deleteLastModifiedRecord() {
+        findElement(By.xpath("//tbody/tr[1]//span/span[contains(.,\"Show Actions\")]"
+                + "/preceding-sibling::span")).click();
+        dropDownMenu = getDriver().findElement(By.xpath("//tbody/tr[1]//a[@title="
+                + "\"Show 2 more actions\"]/ancestor::div[@id and @data-interactive-uid]"));
+        clickOnARecordDropDownMenuDelete();
+    }
+
+    /**
+     * Returns an alert message.

```

Comment:
Uniform comment

Reply:
Check difference between method getDeletedSuccessMessage and recentlyViewedSpanVisible